### ISSUE TITLE
Motor Distance and Position

### ISSUE THREAD

**Issue by codeadamca** [WasabiFan/ev3dev-lang-js#24]
Whenever I use the runForDistance, runToAbsolutePosition, runToPosition or runToRelativePosition methods on the Motor or LargeMotor class, my motor locks up, as in I can't even turn it by hand. If I reset it, it frees up. 

The runForTime and runForever work fine. 

I don't get any visual errors. 

It does actually appear to move slightly. I monitored the position value just using "cat" at the command line, and it went from like 100 to 99. 

Any help would be greatly appreciated!

**Comment by WasabiFan** [comment]
Try running the motors via the command line by directly prodding the drivers, as described here: http://www.ev3dev.org/docs/tutorials/tacho-motors/. That'll indicate if it is a library problem or a driver/motor problem. Also, please post your code so I can see what you are calling.

It sounds like the `hold` stop command, but I'm not sure why it wouldn't move at all beforehand.

**Comment by codeadamca** [comment]
I tested out the motors using the command line and everything seemed to work fine. So I revisited my code and it ends up that the speed I was specifying was really low (around 0.2), my mistake. Once I gave that a number (between 0 and 1050) it worked fine. Thanks for your help. 

I spend a long time trying to get that to work, I don't know how I didn't notice that after a few hours. 

Thanks again!
